py/sight/sight.py

```diff
@@ -738,6 +742,33 @@ def get_attribute(self, key: str) -> str:
       return ''
     return values[-1]
 
+  def _upload_avro_file_to_gcs(self):
+    self.avro_file_counter += 1
+    upload_blob_from_stream(
+        self.params.bucket_name,
+        self.params.gcp_path,
+        self.avro_log,
+        self.avro_log_file_path,
+        self.avro_file_counter,
+    )
+    if self.avro_file_counter == 1:
+      create_external_bq_table(self.params, self.table_name, self.id)
+      logging.info(
```

Comment:
I know I always demand we use logging.info but in this case we should use print since this is meant as a user-facing message.